src/targets/__tests__/crates.test.ts

```diff
@@ -18,14 +18,25 @@ function cratePackageToDependency(cratePackage: CratePackage): CrateDependency {
   return {
     name: cratePackage.name,
     req: '1.0.0',
+    kind: null,
+  };
+}
+
+function makeDev(dependency: CrateDependency): CrateDependency {
+  return {
+    ...dependency,
+    kind: 'dev',
   };
 }
 
 describe('getPublishOrder', () => {
   process.env.CRATES_IO_TOKEN = 'xxx';
-  const target = new CratesTarget({}, new NoneArtifactProvider());
+  const target = new CratesTarget(
+    { noDevDeps: true },
```

Comment:
nit: would be nice to add an interface for CratesTargetConfig (since it's not automatically generated now).

Reply:
Do you have an example for where to put and enforce this? I looked into the other targets, but they all receive an `any` config.

Reply:
They do, but then we perform manual validation, e.g. https://github.com/getsentry/craft/blob/master/src/targets/npm.ts#L131-L158

Reply:
Excellent, thanks. Added.